sw/device/silicon_creator/rom/rom.c

```diff
@@ -219,16 +219,16 @@ static rom_error_t rom_verify(const manifest_t *manifest,
   HARDENED_RETURN_IF_ERROR(sigverify_rsa_key_get(
       sigverify_rsa_key_id_get(&manifest->rsa_modulus), lc_state, &rsa_key));
 
-  const sigverify_spx_key_t *spx_key;
```

Comment:
nit: I'm curious if these initializations are required or not. If not, I'd prefer skipping them. It seems like they are not required because `rsa_key` above is not initialized.